examples/activities/resample.js

```diff
@@ -62,19 +69,28 @@ module.exports = function resample(localInputPath, outputUri, options, callback)
         return done.apply(null, arguments);
       }
 
-      var ds = gdal.open(localOutputPath);
-
-      try {
-        ds.bands.get(1).getStatistics(false, true);
-      } catch (err) {
-        // likely "Failed to compute statistics, no valid pixels found in
-        // sampling.", which means that it's all NODATA, and thus not worth
-        // saving
-
-        return done();
-      } finally {
-        ds.close();
-      }
+      // try {
```

Comment:
For now, should I revert back to doing it, delete the code, or comment it out with a TODO saying that we need to find a way to filter ND tiles?